bootstrap.sh

```diff
@@ -2,15 +2,11 @@
 
 set -eu -o pipefail
 
-if [ -f init.sh ]; then
-    source init.sh
```

Comment:
There was a reason for having the 2 different options but I don't recall what this was at the moment... It might have just been for running locally for testing.  If this is the case then I can add logic for a more explicit testing mode.  I'll add this if/when it is needed.